components/playlist/browser/resources/playerEventSink.ts

```diff
@@ -25,36 +27,64 @@ function handlePlayerEvents (payload: PlayerEventsPayload) {
         payload.data.lastPlayedPosition
       )
       break
-    
+
     case types.PLAYLIST_GO_BACK_TO_CURRENTLY_PLAYING_FOLDER: {
       if (history) {
-        const {currentList, currentItem } = payload.data
+        const { currentList, currentItem } = payload.data
         history.push(`/playlist/${currentList.id}#${currentItem.id}`)
       }
-      break;
+      break
     }
 
     case types.PLAYLIST_OPEN_SOURCE_PAGE: {
-      window.open(
-        payload.data.pageSource.url,
-        '_blank',
-        'noopener noreferrer'
+      window.open(payload.data.pageSource.url, '_blank', 'noopener noreferrer')
```

Comment:
I think this should `break` - the open will be in a new tab so I think we'll fall through to the next case.